src/sink/null.rs

```diff
@@ -35,11 +34,11 @@ impl Sink<NullConfig> for Null {
         Valve::Open
     }
 
-    fn deliver(&mut self, _: sync::Arc<Option<Telemetry>>) -> () {
+    fn deliver(&mut self, _: Telemetry) -> () {
```

Comment:
This and `delivery_line` can be removed with the new default impls in the Sink trait.